frontend/build/css/main.css

```diff
@@ -4659,6 +4655,9 @@ body.admin .top_bar {
 li.rap_detail {
   margin-bottom: 10px;
 }
+pre.address {
+  overflow: visible;
```

Comment:
Not necessary, should either not use the `pre` tag (preserves whitespace) or should abbreviate lines.